File: lab2-semester1/2.cpp
//C++  Program to Calculate Area of Scalene Triangle

#include<iostream>
using namespace std;

int main(){
    float b, h, area;

    cout<<"Enter lenght of base: ";
    cin>>b;
    
    cout<<"Enter lenght of height: ";
    cin>>h;
    
    area=(h*b)/2;
    
      cout<<"\nArea of Scalene Triangle = "<<area<<endl;
}

File: lab2-semester1/2.go
//Program to Calculate Area of Scalene Triangle

package main

import "fmt"

func main() {
    var b, h, area float64
    
    fmt.Print("Input lenght of base: ")
    fmt.Scanln(&b)//input from user
    
    fmt.Print("Input lenght of height: ")
    fmt.Scanln(&h)//input from user
    
    area=(h*b)/2 //formula to calculate area of triangle
    
    fmt.Println("Area: ", area)
}
